### ISSUE TITLE
Failed to Install

### ISSUE THREAD

**Issue by paperclone22** [rockerbacon/lutris-skyrimse-installers#58]
Hi,

I'm trying to install Mod Organizer 2 using your script for Skyrim legendary edition and Steam Proton. Can you help with this?

using default filepaths for steam and Skyrim
Glorious Eggroll version Proton-5.4-GE-3

My setup:
OS: Ubuntu 19.10 x86_64 
Kernel: 5.6.11-050611-generic 
Shell: bash 5.0.3 
Resolution: 1920x1080 
DE: KDE 
WM: KWin 
Terminal: konsole 
CPU: AMD Ryzen 5 3600 6- (12) @ 4.199GHz 
GPU: NVIDIA NVIDIA Corporation Device 1f06 
Memory: 3627MiB / 16020MiB

logs from running the provided command: lutris -i modorganizer2.yml
[lutris-mo2-installer-2.6.7-stable.log](https://github.com/rockerbacon/lutris-skyrimse-installers/files/4640683/lutris-mo2-installer-2.6.7-stable.log)

**Comment by rockerbacon** [comment]
Mod Organizer 2 does not work with Proton GE. As of now, only Steam's Proton 5.0 seems to support the vcruntime it uses.
Any reason why you want to use Proton GE? Everything was setup to work out of the box without the need for any custom build of Proton.

Line 414:
> wine cmd.exe /c echo '%AppData%' returned empty string, error message ""

This indicates that you have used two incompatible versions of Wine in the same prefix. I recommend you purge your current prefix and let Steam setup a new one:

1. Remove (or rename) the directory `$HOME/.steam/steam/compatdata/72850`
2. Configure Skyrim to use Proton 5 in Steam
3. Launch Skyrim on Steam. No need to play the game here, just wait for the Launcher to show up

After that everything should work, remember that you'll need to reinstall Mod Organizer 2.

**Comment by paperclone22** [comment]
didn't have any luck yet.

like you said, I removed the directory: /home/sbm/.steam/steam/steamapps/compatdata/72850
, set Skyrim to use Proton 5.0-7
and started Skyrim launcher

I still get the error: wine cmd.exe /c echo '%AppData%' returned empty string, error message ""
 any other ideas?

**Comment by paperclone22** [comment]
An interesting note, if I run the errored command with the exact path to Proton 5 wine, then it does show the AppData folder.

WINEPREFIX=/home/sbm/.steam/steam/steamapps/compatdata/72850/pfx; "/home/sbm/.steam/steam/steamapps/common/Proton 5.0/dist/bin/wine" cmd.exe /c echo '%AppData%'

> wineserver: using server-side synchronization.
0034:err:module:__wine_process_init L"C:\\windows\\system32\\winemenubuilder.exe" not found
002c:err:wineboot:process_run_key Error running cmd L"C:\\windows\\system32\\winemenubuilder.exe -a -r" (126).
006c:err:module:load_so_dll failed to load .so lib "/home/sbm/.steam/steam/steamapps/common/Proton 5.0/dist/bin/../lib64/wine/winebus.sys.so": libudev.so.0: cannot open shared object file: No such file or directory
006c:err:ntoskrnl:ZwLoadDriver failed to create driver L"\\Registry\\Machine\\System\\CurrentControlSet\\Services\\winebus": c0000142
003c:fixme:service:scmdatabase_autostart_services Auto-start service L"winebus" failed to start: 1114
0084:err:ntoskrnl:ZwLoadDriver failed to create driver L"\\Registry\\Machine\\System\\CurrentControlSet\\Services\\wineusb": c0000142
003c:fixme:service:scmdatabase_autostart_services Auto-start service L"wineusb" failed to start: 1114
C:\users\steamuser\Application Data

The previous behaviour shows up again if i just use the wine version installed at /usr/bin/wine . aka. wine-5.8 (Staging)

WINEPREFIX=/home/sbm/.steam/steam/steamapps/compatdata/72850/pfx; wine cmd.exe /c echo '%AppData%'

> 002c:err:module:__wine_process_init failed to load L"C:\\windows\\system32\\wineboot.exe", error c0000018
0024:err:environ:run_wineboot failed to start wineboot c0000018
0034:err:module:__wine_process_init failed to load L"C:\\windows\\system32\\explorer.exe", error c0000018
0024:fixme:winediag:start_process Wine Staging 5.8 is a testing version containing experimental patches.
0024:fixme:winediag:start_process Please mention your exact version when filing bug reports on winehq.org.
%AppData%

**Comment by paperclone22** [comment]
I got MO2 installed now! Yay!

I did a few things that eventually got the install to work.

1. I changed the wine config in the lutris gui to point to where Proton 5 in located: "/home/sbm/.steam/steam/steamapps/common/Proton 5.0/dist/bin/wine" . and I toggled "use system winetricks" on
2. I purged wine staging: `sudo apt purge wine-staging`
3. I got a error "wineserver not found!" so I reinstalled winetricks

that did it. I re-ran the install command and it all worked. Thanks for the help and the cool script. MO2 is my favorite mod manger
